fava/api/__init__.py

```diff
@@ -166,6 +166,9 @@ def _journal(self, postings, include_types=None, with_change_and_balance=False):
 
             entry = serialize_entry(posting)
 
```

Comment:
I'll move it to `serialize_entry`. Adding a filter for this one use seems a bit over-kill to me...